tests/api/ils/test_metadata_extensions.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test metadata extensions."""
+
+import pytest
+from invenio_indexer.api import RecordIndexer
+from invenio_records_rest.schemas.fields import DateString, SanitizedUnicode
+from marshmallow import ValidationError
+from marshmallow.fields import Bool, Integer, List
+
+from invenio_app_ils.records.metadata_extensions import MetadataExtensions
+
+
+@pytest.fixture(scope='module')
+def app_config(app_config):
```

Comment:
is this then applied everywhere? Otherwise you can simply create a new app fixture with this config: https://github.com/inveniosoftware/invenio-app-ils/blob/352551dfbdab4332920d59e019a9ebee229f9570/tests/api/conftest.py#L230